<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1585</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameContentAfterImport">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1301</width>
      <height>851</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableChannel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>580</y>
       <width>421</width>
       <height>241</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Channel G</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channel H</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channel S</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channel V</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBoxImages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>571</height>
      </rect>
     </property>
     <property name="title">
      <string>Imagens</string>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>491</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="originalTab">
       <attribute name="title">
        <string>Original</string>
       </attribute>
       <widget class="QGraphicsView" name="originalImageView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="originalImageLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>441</width>
          <height>391</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="greyscaleTab">
       <attribute name="title">
        <string>Grayscale</string>
       </attribute>
       <widget class="QGraphicsView" name="grayscaleImageView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>461</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="greyscaleImageLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>19</y>
          <width>441</width>
          <height>391</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="zoomInButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>490</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zoomOutButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom out</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextImageButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>351</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Image</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupMatrixAll">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>20</y>
       <width>351</width>
       <height>821</height>
      </rect>
     </property>
     <property name="title">
      <string>Co-occurrence Matrix </string>
     </property>
     <widget class="QGroupBox" name="groupMatrix04_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (01x01)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_6">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix01ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupMatrix04">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (02x02)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix02ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupMatrix04_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (04x04)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix04ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupMatrix04_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (08x08)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix08ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupMatrix04_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (16x16)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix16ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupMatrix04_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>680</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Matrix (32x32)</string>
      </property>
      <widget class="QTableWidget" name="tableMatrix05_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>201</width>
         <height>101</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>Contrast</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Entropy</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Homogeneity</string>
        </property>
       </row>
      </widget>
      <widget class="QFrame" name="frameMatrix03_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="imageMatrix03_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="matrix32ImageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>101</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableChannel_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>620</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <row>
      <property name="text">
       <string>Original</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>EFF NET 2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>EFF NET 6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>XGBOOST</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>XGBOOST 6</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Classificador</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="grayscaleHistogram" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>381</width>
       <height>261</height>
      </rect>
     </property>
     <widget class="QLabel" name="grayscaleHistogramLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="hsvHistogram" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>381</width>
       <height>261</height>
      </rect>
     </property>
     <widget class="QLabel" name="hsvHistogramImageLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1585</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionDataset"/>
   </widget>
   <addaction name="menuImport"/>
  </widget>
  <action name="actionDataset">
   <property name="text">
    <string>Dataset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
